libs/api/domains/rsk/src/lib/api-domains-rsk.resolver.ts

```diff
@@ -0,0 +1,22 @@
+import { Query, Resolver } from '@nestjs/graphql'
+import { UseGuards } from '@nestjs/common'
+
+import {
+  IdsAuthGuard,
+  ScopesGuard,
+  CurrentUser,
+  User,
+} from '@island.is/auth-nest-tools'
+import { RSKService } from '@island.is/clients/rsk'
+import { CurrentUserCompanies } from './models/currentUserCompanies.model'
+
+@UseGuards(IdsAuthGuard, ScopesGuard)
+@Resolver()
+export class RSKResolver {
+  constructor(private RSKService: RSKService) {}
+
+  @Query(() => [CurrentUserCompanies])
+  async rskGetCurrentUserCompanies(@CurrentUser() user: User) {
```

Comment:
Kinda picky from me, but I believe you won't use the `get` keyword to follow our graphql naming convention. Just straight `rskCurrentUserCompanies`

https://docs.devland.is/handbook/technical-overview/api-design-guide/graphql-naming-conventions#example

Reply:
Ty, will update this to match naming convention